packages/riipen-ui-docs/src/pages/components/paper/Outline.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import Paper from "riipen-ui/components/Paper";
+
+export default function Outline() {
+  const style = {
+    marginRight: "20px"
+  };
+
+  return (
+    <React.Fragment>
+      <Paper
+        variant="outlined"
+        style={{
+          height: "100px",
+          width: "100px"
+        }}
+      />
+
+      <span style={style} />
+
+      <Paper
+        variant="outlined"
+        rounded={true}
+        style={{
+          height: "100px",
+          width: "100px"
+        }}
+      />
```

Comment:
Both of these examples are showing outlined. I think in this file we should show one outlined and one not outlined, and that we should actually move the rounding / not rounding to a different file to show those off in a separate section. Thoughts? I think it would be just nice from a user standpoint to see those things separately.

Reply:
I thought about it too, but it's pretty hard to see the rounded corner without the outline. I've adjusted it for now so you can have a look. Maybe the color of the background can be adjusted just in `Rounded.jsx`

Reply:
I just pulled but don't see the update. Maybe you haven't pushed yet and I am too eager :P.